src/tagmanager-wrap.js

```diff
@@ -0,0 +1,80 @@
+
+const defaultOptions = {
+  virtualPageViewEvent: 'virtual_pageview',
+  startPush: {
+    experiements: [],
+  },
+  addExperiments: [],
+};
+
+export default class TagManager {
+  constructor(dataLayer, params = defaultOptions) {
+    this.options = Object.assign(defaultOptions, params);
+    this.dataLayer = dataLayer || [];
+    this.addExperiments();
+    this.dataLayer.push(this.options.startPush);
+  }
+
+  addExperiments() {
+    Object.assign(this.options.startPush, {
+      experiments: this.experiments(),
+    });
+
+    return this;
+  }
+
+  experiments() {
+    return this.options.startPush.experiments.concat(this.options.addExperiments);
+  }
+
+  prependExperiment(experiment) {
+    if (this.dataLayer[0].experiments) {
+      const dataLayer = Object.assign({}, this.options.startPush, {
+        experiments: [experiment],
+      });
+
+      this.dataLayer.unshift(dataLayer);
+      return this.dataLayer[0].experiments;
+    }
+
+    return this.dataLayer[0].experiments.push(experiment);
+  }
+
+  virtualPageView(vpname, event) {
+    this.dataLayer.push({
+      vpname,
+      event: event || this.options.virtualPageViewEvent,
+    });
+  }
+
+  eventCategory(event, infos) {
+    const categoryObj = Object.assign({
+      eventCategory: event,
+      event: 'GAEvent',
+    }, infos || {});
+
+    return this.dataLayer.push(categoryObj);
+  }
+
+  custom(obj) {
+    return this.dataLayer.push(obj);
+  }
+
+  bindEvents($) {
```

Comment:
@iondrimba I'm thinking about to control this re-bind, storing elements on some array, in TagManager class. jQuery is a dependency only because event namespaces (`on("click:ga-event")`), but he controls this, storing handlers on dom object.

I think that storing the elements solves this issue and we can remove this dependency.